package.json

```diff
@@ -13,21 +13,21 @@
   "license": "Talk to the Legal Department",
   "main": "NestHydrationJS.js",
   "scripts": {
-    "coverage": "node_modules/istanbul/lib/cli.js cover node_modules/jasmine/bin/jasmine.js",
```

Comment:
Odd, ran locally fine for me. Perhaps it was an npm version problem. I'll change it back.